console/src/components/Services/ApiExplorer/Actions.js

```diff
@@ -44,6 +44,20 @@ const CREATE_WEBSOCKET_CLIENT = 'ApiExplorer/CREATE_WEBSOCKET_CLIENT';
 const FOCUS_ROLE_HEADER = 'ApiExplorer/FOCUS_ROLE_HEADER';
 const UNFOCUS_ROLE_HEADER = 'ApiExplorer/UNFOCUS_ROLE_HEADER';
 
+let WEBSOCKET_SUBSCRIPTION_CLIENT;
```

Comment:
We usually use the upper case for some string constants (as above). Let's just use the camel case here.